README.md

```diff
@@ -25,6 +26,7 @@ The `setup-gazebo` GitHub Action sets up an environment to install a Gazebo rele
 - `required-gazebo-distributions`: A **required** parameter that specifies the Gazebo distribution to be installed.
 - `use-gazebo-prerelease`: An **optional** parameter to install pre-release binaries from OSRF repository.
 - `use-gazebo-nightly`: An **optional** parameter to install nightly binaries from OSRF repository.
+- `install-ros-gz`: An **optional** parameter to install the ROS 2 Gazebo bridge (`ros_gz`).
```

Comment:
```suggestion
- `install-ros-gz`: An **optional** parameter to install the ROS 2 Gazebo bridge (`ros_gz`). This will require a previous ROS installation (i.e. using [ros-setup](https://github.com/ros-tooling/setup-ros). The installation of the `ros_gz` bridge supports the ROS official and ROS non official (from packages.osrfoundation.org) variants following the [Installing Gazebo with ROS](https://gazebosim.org/docs/ionic/ros_installation/#summary-of-compatible-ros-and-gazebo-combinations)  documentation.)
```

Reply:
let's provide a bit more of information

Reply:
Added this information to the README